Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,472 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Globalization;
+using System.Text;
+using System;
+using System.Diagnostics;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        public static string defaultCurrencySymbol = "¤";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'defaultCurrencySymbol' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971928)